# Key states for dragon robot trajectory
# Each key state:  root link pose + 6 joint angles
# [x, y, z, roll, pitch, yaw, joint1, joint2, joint3, joint4, joint5, joint6]

key_states:
  - [0.5, 0.5, 1.0, 0.0, 0.0, 3.14, 0.6, 1.57, 0.6, 1.57, -1.0, 1.57]
  - [0.6, 0.0, 1.0, 0.0, 0.0, 1.57, 0.0, -1.57, 0.0, 1.57, 0.0, 1.57]
  - [0.5, -0.5, 1.0, 0.0, 0.0, 0.0, -0.6, -1.57, -0.6, -1.57, 1.0, 1.57]
  - [0.0, 0.0, 1.0, 0.0, 0.0, -0.6, 0.0, -1.57, 0.0, -1.57, 0.0, -1.57]
  - [-0.5, 0.5, 1.0, 0.0, 0.0, 0.0, 0.6, 1.57, 0.6, -1.57, -1.0, -1.57]
  - [-0.6, 0.0, 1.0, 0.0, 0.0, 1.57, 0.0, 1.57, 0.0, 1.57, 0.0, -1.57]
  - [-0.5, -0.5, 1.0, 0.0, 0.0, 3.14, -0.6, 1.57, -0.6, 1.57, 1.0, 1.57]
  - [0.0, 0.0, 1.0, 0.0, 0.0, 3.94, 0.0, 1.57, 0.0, 1.57, 0.0, 1.57]
